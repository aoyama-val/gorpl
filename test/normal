# 通常
hoge
moge
hoge hoge
