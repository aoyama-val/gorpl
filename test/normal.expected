# 通常
fuga
moge
fuga fuga
